llm_on_ray/finetune/DataPreprocess.py

```diff
@@ -0,0 +1,224 @@
+#
+# Copyright 2023 The LLM-on-Ray Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import copy
+import re
+from itertools import chain
+
+import torch
+
+IGNORE_INDEX = -100
+
+
+class AlpacaDataPreprocess:
```

Comment:
does this class only used for preprocessing alpaca dataset?

Reply:
Yes, In this PR, it just preprocess alpaca dataset. But in DPO PR, it will be add DPOIntelOrcaPreprocesser class to preprocess intel orca dataset.